.github/workflows/build.yml

```diff
@@ -35,18 +35,14 @@ jobs:
 
     - name: Checkout
       uses: actions/checkout@v2
-    - name: Setup dotnet 2.1
+    - name: Setup dotnet 2.1 3.1 5.0 6.0
       uses: actions/setup-dotnet@v1
       with:
-        dotnet-version: '2.1.x'
-    - name: Setup dotnet 3.1
-      uses: actions/setup-dotnet@v1
-      with:
-        dotnet-version: '3.1.x'
-    - name: Setup dotnet 5.0
-      uses: actions/setup-dotnet@v1
-      with:
-        dotnet-version: '5.0.x' 
+        dotnet-version: |
+          2.1.x
+          3.1.x
```

Comment:
Let's remove .NET Core 2.1 SDK from here as it is no longer supported.

Reply:
Removed